src/Compilers/CSharp/Portable/Compilation/CSharpCompilation.cs

```diff
@@ -1064,6 +1066,18 @@ internal new ModuleSymbol SourceModule
             }
         }
 
+        internal VariableUsePass VariableUsePass
+        {
+            get
+            {
+                if (_lazyVariableUsePass == null)
+                {
+                    Interlocked.CompareExchange(ref _lazyVariableUsePass, new VariableUsePass(this.SourceAssembly), null);
```

Comment:
>Interlocked.CompareExchange(ref _lazyVariableUsePass, new VariableUsePass(this.SourceAssembly), null); [](start = 20, length = 102)

We probably do not care about the identity of the VariableUsePass instance. Therefore, interlocked exchange is not necessary, we can simply assign to the field. The code as it is is fine though.